test/unit/model/define.test.js

```diff
@@ -35,15 +35,60 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         current.define('foo', {
           id: DataTypes.INTEGER,
         });
-      }).to.throw('A column called \'id\' was added to the attributes of \'foos\' but not marked with \'primaryKey: true\'');
+      }).to.throw('An attribute called \'id\' was defined in model \'foos\' but not marked as a primaryKey. This is likely to be an error, which can be fixed by setting its \'primaryKey\' option to true. If this is intended, explicitly set its \'primaryKey\' option to false');
 
       expect(() => {
         current.define('bar', {
           id: {
             type: DataTypes.INTEGER,
           },
         });
-      }).to.throw('A column called \'id\' was added to the attributes of \'bars\' but not marked with \'primaryKey: true\'');
+      }).to.throw('An attribute called \'id\' was defined in model \'bars\' but not marked as a primaryKey. This is likely to be an error, which can be fixed by setting its \'primaryKey\' option to true. If this is intended, explicitly set its \'primaryKey\' option to false');
+    });
+
+    it('should allow model definition without PK', () => {
+      const Model = current.define('User', {}, {
```

Comment:
nit: avoid naming Models 'Model' to disambiguate with the 'Model' class provided by Sequelize

```suggestion
      const User = current.define('User', {}, {
```